app/assets/scripts/components/card.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { PropTypes as T } from 'prop-types';
+import styled, { css } from 'styled-components';
+
+const CardWrapper = styled.article`
+  ${({gridColumn, gridRow}) => css`
+    grid-column: ${gridColumn || 'auto'};
+    grid-row: ${gridRow || 'auto'};
+  `}
+`;
+
+const CardContents = styled.a`
+`;
+
+const CardHeader = styled.header`
+`;
+
+const CardBody = styled.div`
+`;
+
+const CardFooter = styled.footer`
+`;
+
+const CardHeadline = styled.div`
+`;
+
+const CardTitle = styled.h1`
+`;
+
+export const CardSubtitle = styled.p`
+`;
+export const HighlightText = styled.h1`
+  font-size: ${({size}) => {
+    switch (size) {
+      case 'small':
+        return '1rem';
+      case 'medium':
+        return '2rem';
+      case 'large':
+        return '3rem';
+      default:
+        return '1rem';
+    }
+  }};
+`;
+
+function Card (props) {
+  const {
+    gridColumn,
+    gridRow,
+    title,
+    subtitle,
+    renderHeader,
+    renderBody,
+    renderFooter
+  } = props;
+  return (
+    <CardWrapper
+      className='card' i
```

Comment:
the  `i` looks accidental?
```suggestion
      className='card'
```